lib/error/error_manager_impl.cpp

```diff
@@ -51,19 +56,25 @@ std::list<ErrorPtr> ErrorManagerImpl::clear_error(const ErrorType& type, const b
         return clear_error(type, sub_type);
     }
     if (!can_be_cleared(type)) {
-        throw EverestArgumentError("Errors can't be cleared, becauce type " + type + " is not active.");
+        EVLOG_warning << "Errors can't be cleared, because type " << type << " is not active.";
+        return {};
     }
     std::list<ErrorFilter> filters = {ErrorFilter(TypeFilter(type))};
     std::list<ErrorPtr> res = database->remove_errors(filters);
+    std::stringstream ss;
+    ss << "Cleared " << res.size() << " errors of type " << type << " with sub_types:" << std::endl;
     for (const ErrorPtr error : res) {
         this->publish_cleared_error(*error);
+        ss << "  - " << error->sub_type << std::endl;
     }
+    EVLOG_info << ss.str();
     return res;
 }
 
 std::list<ErrorPtr> ErrorManagerImpl::clear_error(const ErrorType& type, const ErrorSubType& sub_type) {
     if (!can_be_cleared(type, sub_type)) {
-        throw EverestArgumentError("Error can't be cleared, because it is not active.");
+        EVLOG_warning << "Error can't be cleared, because it is not active.";
```

Comment:
Probably also "Error**s** can't", as in the other implementation of `clear_error()`.

This should also print type and subtype:

```suggestion
        EVLOG_warning << "Errors can't be cleared, because type " << type << ", subtype " << sub_type << " is not active.";
```

Reply:
> Probably also "Errors can't", as in the other implementation of clear_error().

In this case it is actually a function to exactly clear one error, the other functions can clear multiple ones

> This should also print type and subtype:

Good point, thanks! I added it